src/App.js

```diff
@@ -2,43 +2,45 @@ import "./App.css";
 import logo from "./logo.svg";
 import { database } from "./firebase";
 import { set, ref, push, get, onValue } from "firebase/database";
-import { useEffect, useState } from "react";
+import { useEffect } from "react";
+import useQuestionBank from "./Hooks/useQuestionBank";
 import { useNavigate } from "react-router-dom";
 
 function App() {
   // const [questionsObject, setQuestionsObject] = useState({});
   const navigate = useNavigate();
-  const [questionsQueue, setQuestionsQueue] = useState([]);
-  /*
-  // Create initial testing questions
-
-  const questionRef = ref(database, process.env.REACT_APP_DB_QUESTIONS_KEY);
-  const currentDate = new Date().toJSON();
-  push(questionRef, {
-    content: "hey this is a scam",
-    is_scam: true,
-    num_users_attempted: 0,
-    num_users_correct: 0,
-    date_created: currentDate,
-  });
-  push(questionRef, {
-    content: "good news this isn't a scam",
-    is_scam: false,
-    num_users_attempted: 0,
-    num_users_correct: 0,
-    date_created: currentDate,
-  });
-  */
+  const { setQuestionBank } = useQuestionBank();
 
-  // Docs: https://firebase.google.com/docs/database/web/read-and-write
+  // Create initial testing questions
+  // const seedData = () => {
+  //   const questionRef = ref(database, process.env.REACT_APP_DB_QUESTIONS_KEY);
+  //   const currentDate = new Date().toJSON();
+  //   push(questionRef, {
+  //     content: "hey this is a scam",
+  //     is_scam: true,
+  //     num_users_attempted: 0,
+  //     num_users_correct: 0,
+  //     date_created: currentDate,
+  //     bin_id: 1,
+  //   });
+  //   push(questionRef, {
+  //     content: "good news this isn't a scam",
+  //     is_scam: false,
+  //     num_users_attempted: 0,
+  //     num_users_correct: 0,
+  //     date_created: currentDate,
+  //     bin_id: 1,
+  //   });
+  //   // ... and more
+  //   // Docs: https://firebase.google.com/docs/database/web/read-and-write
+  // }
```

Comment:
Shall we move this to its own file?

Reply:
When we have the dataset from OGP, it would be a good idea to have a separate file that seeds data.